litrpc/proxy.proto

```diff
@@ -0,0 +1,22 @@
+syntax = "proto3";
+
+package litrpc;
+
+option go_package = "github.com/lightninglabs/lightning-terminal/litrpc";
+
+service Proxy {
+    rpc Ping (PingRequest) returns (PingResponse);
```

Comment:
Should we instead add a `GetInfo` call? That currently just returns the version (or maybe all the info printed in `showStartupInfo()`? Or will that overlap too much with the status server? (need to reload the context on that).